guides/theming.md

```diff
@@ -367,6 +367,83 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Strong focus indicators
+
+By default, most components indicate browser focus by changing their background color as prescribed
+by the Material Design specification. This behavior, however, can fall short of accessibility
```

Comment:
My big question after reading these docs is, when _can_ it fall short, and if it does, why wouldn't I just always enable strong focus indication if it's a known shortcoming? When would I require a stronger indication of browser focus? Maybe just a visual example of what this looks like would be helpful

Reply:
IIRC WCAG 2.2 implies that strong focus indicators are necessary but doesn't outright say it. Google'e internal requirements don't require it today, but probably will in the next year or so. We don't do this by default because it's not included in the Material Design spec. I've communicated to MD that it's a priority for us, but haven't seen any movement on it yet.

Reply:
Ok, makes sense this is in the main theming guide then and likely should be built in!